runtime/compiler/p/codegen/J9TreeEvaluator.hpp

```diff
@@ -86,6 +90,9 @@ class OMR_EXTENSIBLE TreeEvaluator: public J9::TreeEvaluator
    static TR::Register *ardbarEvaluator(TR::Node *node, TR::CodeGenerator *cg);
    static TR::Register *ardbariEvaluator(TR::Node *node, TR::CodeGenerator *cg);
    
+   static TR::Register *aConcurrentScavengeSequence(TR::Node *node, TR::CodeGenerator *cg);
+   static TR::Register *iConcurrentScavengeSequence(TR::Node *node, TR::CodeGenerator *cg);
```

Comment:
Are these declarations needed? I believe we only use these inside `J9TreeEvaluator.cpp`. Unless there's a reason why they need to be declared here or be public member functions.